monitoring/base/alertmanager.yaml

```diff
@@ -28,6 +28,6 @@ kind: Alertmanager
 metadata:
   name: thoth-monitoring
 spec:
-  externalUrl: "http://alert-manager-thoth-infra-prod.apps.zero.massopen.cloud/"
+  externalUrl: ""
```

Comment:
why this is empty?

Reply:
we wanted to keep the `base` contain separate from direct cluster-related values, so the value is embedded in the overlay.
https://github.com/thoth-station/thoth-application/blob/a0e1d0fcaef73ce8b72e3d77c2ffd3255bc2ab7e/monitoring/overlays/cnv-prod/kustomization.yaml#L20